services/docker/generate.go

```diff
@@ -75,6 +75,14 @@ func MarshalledDefaults() []byte {
 		log.Fatalf("error: %v", err)
 	}
 
+	// Append the volume setting on to the docker-compose setting directly
+	mysqlVolName := "tok_" + conf.GetConfig().Tokaido.Project.Name + "_mysql_database"
```

Comment:
@yjimk see here for workaround to having dynamic "name" entry for "volumes" struct, as mentioned in Slack. 

I'm not super excited about this, but in lieu of a better workaround, I'd like to get this out in tomorrow's release as it's a _huge_ improvement.

Reply:
Yeah, happy to stick with what works for the moment as it's hard to judge the limitation from this PR. Let's just have a card to address your concerns for the future